usermanagerlib/src/test/java/org/pesmypetcare/usermanagerlib/clients/GoogleCalendarManagerClientTest.java

```diff
@@ -0,0 +1,157 @@
+package org.pesmypetcare.usermanagerlib.clients;
+
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnit;
+import org.mockito.junit.MockitoRule;
+import org.pesmypetcare.usermanagerlib.datacontainers.DateTime;
+import org.pesmypetcare.usermanagerlib.datacontainers.EventData;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.ExecutionException;
+
+import static junit.framework.TestCase.assertEquals;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.BDDMockito.given;
+
+/**
+ * @author Marc Simó
+ */
+public class GoogleCalendarManagerClientTest {
+    private static final String ACCESS_TOKEN = "my-token";
+    private static final StringBuilder STATUS_OK = new StringBuilder("200");
+    private static final int STATUS_OK_INT = 200;
+    private static final String SHOULD_RETURN_200 = "Should return response code 200";
+
+    private String owner;
+    private String petName;
+    private EventData eventData;
+    private List<EventData> eventDataList;
+    private String eventId;
+    private StringBuilder jsonEventData;
+    private StringBuilder jsonAllEvents;
+
+    @Mock
+    private TaskManager taskManager;
+
+    @InjectMocks
+    private GoogleCalendarManagerClient client = new GoogleCalendarManagerClient();
+
+    @Rule
+    public MockitoRule mockitoRule = MockitoJUnit.rule();
+
+    @Before
+    public void setUp() {
+        owner = "Manolo";
+        petName = "Kawaguchi";
+        DateTime date = DateTime.Builder.buildFullString("2020-02-13T10:30:00");
+        DateTime date2 = DateTime.Builder.buildFullString("2020-02-13T12:30:00");
+        eventId = "eventId";
+        eventData = new EventData("eventId", "My summary", "My location",
+            "My description", EventData.BANANA, 80, 14,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['14' is a magic number.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Libraries/pullRequest?prid=5405224)